### PR TITLE
options/windowList: Add event names to alert level tips

### PR DESCRIPTION
**Pull request by staticfox**
This should make it more clear what each alert level will trigger on

https://i.imgur.com/QXP1MjY.png

### CODE REVIEW

**Comment by AndrioCelos** [comment]
What if you reconfigure them?

File: src/modules/options/OptionsWidget_windowList.cpp [hunk 1]
```diff
@@ -29,6 +29,9 @@
 #define _WANT_OPTION_FLAGS_
 #include "KviOptions.h"
 
+#include <string>
+#include <vector>
```
Comment (AndrioCelos) [inline src/modules/options/OptionsWidget_windowList.cpp:33]: Does this compile?

Comment (staticfox) [reply src/modules/options/OptionsWidget_windowList.cpp:33]: Why wouldn't it?

**Comment by staticfox** [comment]
AV & Travis passed

**Pull request merged**